### PR TITLE
Larry Cashdollar

### PR DESCRIPTION
**Pull request by kurtseifried**
manually entering in https://github.com/distributedweaknessfiling/cvelist/pull/21/files

### CODE REVIEW

File: 2018/1002xxx/CVE-2018-1002001.json [hunk 1]
```diff
@@ -0,0 +1,71 @@
+{
```
Comment (cve-team) [inline 2018/1002xxx/CVE-2018-1002001.json]: Invalid \escape: line 21 column 332 (char 942)

**Comment by cve-team** [comment]
Are you able to fix the JSON so we can populate the entries in the CVE List?

**Comment by lcashdol** [comment]
Yes, I can work on this if I have access.

File: 2018/1002xxx/CVE-2018-1002001.json (hunk 1 above)
Comment (lcashdol) [reply 2018/1002xxx/CVE-2018-1002001.json]: Fixed.

**Comment by lcashdol** [comment]
Can someone take a look at https://github.com/distributedweaknessfiling/cvelist/pull/16

**Comment by cve-team** [comment]
I'm not sure what concerns there are with that. Once it's ready, though, kurtseifried needs to push it to us.

**Pull request merged**